storage/lib/storage_samples.ex

```diff
@@ -20,4 +20,35 @@ defmodule GoogleApi.Storage.Samples do
     Enum.each(response.items, &IO.puts(&1.id))
     "done!"
   end
+
+  # [start storage_upload_file]
+  @doc """
+  List storage buckets for a project.
+
+  ## Examples
+
+      iex> GoogleApi.Storage.Samples.upload_file("bucket_id", "test/file.txt")
+      Uploaded file.text to  https://www.googleapis.com/storage/v1/b/bucket_id/o/file.txt
+      your_project_bucket2
+
+  """
+  def upload_file(bucket_id, file) do
```

Comment:
Maybe to clarify, rename to `file_fullpath` so the reader doesn't think it's some kind of file struct.

Reply:
Relative file paths also work, so how about `file_path`?